app/workers/publish_taxon_worker.rb

```diff
@@ -0,0 +1,10 @@
+class PublishTaxonWorker
+  include Sidekiq::Worker
+
+  def perform(taxon_content_id)
+    Services.publishing_api.publish(taxon_content_id)
+  rescue GdsApi::HTTPClientError => ex
```

Comment:
uh, you could just rescue the `GdsApi::HTTPConflict` error now. No need to inspect the contents of the error message.